deploy/037_deploy_mode_spokepool.ts

```diff
@@ -0,0 +1,35 @@
+import { deployNewProxy, getSpokePoolDeploymentInfo } from "../utils/utils.hre";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { L2_ADDRESS_MAP } from "./consts";
+import { ZERO_ADDRESS } from "@uma/common";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { hubPool, spokeChainId } = await getSpokePoolDeploymentInfo(hre);
+
+  const initArgs = [
+    1,
+    // Set hub pool as cross domain admin since it delegatecalls the Adapter logic.
+    hubPool.address,
+    hubPool.address,
+  ];
+  // Construct this spokepool with a:
+  //    * A WETH address of the WETH address
+  //    * A depositQuoteTimeBuffer of 1 hour
+  //    * A fillDeadlineBuffer of 6 hours
+  //    * Native USDC address on L2
+  //    * CCTP token messenger address on L2
+  const constructorArgs = [
+    "0x4200000000000000000000000000000000000006",
+    3600,
+    21600,
+    ZERO_ADDRESS,
```

Comment:
Does this look right? Mode does not have native USDC, only USDC.e.

Reply:
Yes we should be able to use the ZERO_ADDRESS as long as we enforce that `cctpTokenMessenger` is the ZERO_ADDRESS

Reply:
Great. Will leave it as is.